neutron_plugin/port.py

```diff
@@ -69,6 +71,8 @@ def create(neutron_client, args, **kwargs):
 
             ctx.instance.runtime_properties[FIXED_IP_ADDRESS_PROPERTY] = \
                 _get_fixed_ip(ext_port)
+            ctx.instance.runtime_properties[MAC_ADDRESS_PROPERTY] = \
+                ext_port['mac_address']
```

Comment:
I disagree. `MAC_ADDRESS_PROPERTY` is a key for runtime_properties on a `cloudify.openstack.nodes.Port`. `'mac_address'` is a key in a neturon_client.port dict. They happen to be the same string right now, but they aren't the same thing and changing one should not mean we change the other.

A similar difference exists between `OPENSTACK_ID_PROPERTY` and `'id'` in https://github.com/chriscoomber/cloudify-openstack-plugin/blob/7340d9f40d91274133118af5567b516b32df3b75/neutron_plugin/port.py#L95.

It's possible we want constants for the keys in the neutron_client.port dict, or better yet to import them from the neutron_client module (if they exist), but that hadn't been done for `'id'` and `'name'` so far, so I didn't do it for `'mac_address'` either. I don't think this is necessary.